src/test/java/org/kpmp/auth/UserAuthTest.java

```diff
@@ -0,0 +1,76 @@
+package org.kpmp.auth;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class UserAuthTest {
+
+    private UserAuth testUserAuth;
+
+    @Before
+    public void setUp() throws Exception {
+        testUserAuth = new UserAuth();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        testUserAuth = null;
+    }
+
+    @Test
+    public void testGettersAndSetters() throws Exception {

```

Comment:
This is actually generally bad practice when writing tests.  Each test should be testing one piece of functionality.  The reason it is considered "bad practice" is as soon as one of the assertions fail, the test terminates and you don't know what else is broken.  I am not going to ask you to update this as part of the review, just pointing it out.